runtime.md

```diff
@@ -123,7 +123,7 @@ This operation MUST generate an error if `process` was not set.
 `kill <container-id> <signal>`
 
 This operation MUST [generate an error](#errors) if it is not provided the container ID.
-Attempting to send a signal to a container that is not running MUST have no effect on the container and MUST [generate an error](#errors).
+Attempting to send a signal to a container that is not running or not created MUST have no effect on the container and MUST [generate an error](#errors).
```

Comment:
I'd rather tie this more clearly to the already-well-defined state values.  Something like:

``Attempting to send a signal to a container that is neither [`created` nor `running`](#state) MUST have no effect…``

That's pretty close to what you're suggesting, although it:

* Uses “neither”/“nor” instead of repeating “not”.
* Links to the “State” section for formal definitions of `created` and `running` states.
* Uses backticks to strengthen the “these are formally defined terms” semantics of `running` and `created`.

[1]: https://github.com/opencontainers/runtime-spec/blame/fd0b0b6f7505ba8d24ee1fe59e59ca969b26843a/runtime.md#L21